provider/testdata/data/baked/compactdecimal/short_v1/ar+ar-EG+ar-EG-u-nu-latn+ar-u-nu-latn.rs.data

```diff
@@ -0,0 +1,30 @@
+::icu_compactdecimal::provider::CompactDecimalPatternDataV1 {
```

Comment:
Praise: I like smaller files. IMO, more smaller files is a win over a few big files.